onnxruntime/hosting/server_configuration.h

```diff
@@ -11,6 +11,8 @@
 namespace onnxruntime {
 namespace hosting {
 
+enum Result { ExitSuccess = 1, ExitFailure, ContinueSuccess };
```

Comment:
COOL! Could we have some test cases for failure scenarios?

Reply:
OK. I've added some but don't want to *too* many